--backend=vs2019 EntryPointSymbol incorrect for 'UNICODE' projects

**Describe the bug**
vcxproj files generated by meson have an incorrect EntryPointSymbol for UNICODE applications (ie. those that have an entry point of wmain (console) or wWinMain (gui)).

```
2>MSVCRTD.lib(exe_main.obj) : error LNK2019: unresolved external symbol main referenced in function "int __cdecl invoke_main(void)" (?invoke_main@@YAHXZ)
2>.\main.exe : fatal error LNK1120: 1 unresolved externals
```

Removing EntryPointSymbol from all generated .vcxproj files works around the issue.

Ninja builds do not have this issue.

**To Reproduce**
meson.build:
``` meson
project('wmain', 'c')

executable('main', 'wmain.c')
```
wmain.c:
``` c
#include <stdio.h>

int wmain(int argc, wchar_t *argv[]) {
    printf("Hello, World!\n");
}
```

**Expected behavior**
main.exe should be generated instead of a link error.

**system parameters**
This is a native build, with Visual Studio 2019, on Windows 10, with meson and ninja from the official 0.53.1 MSI (I looked at the code, and it's still the same in master on github).

--- a/mesonbuild/backend/vs2010backend.py
+++ b/mesonbuild/backend/vs2010backend.py
@@ -744,19 +744,16 @@ class Vs2010Backend(backends.Backend):
 
     def gen_vcxproj(self, target, ofname, guid):
         mlog.debug('Generating vcxproj %s.' % target.name)
-        entrypoint = 'WinMainCRTStartup'
         subsystem = 'Windows'
         self.handled_target_deps[target.get_id()] = []
         if isinstance(target, build.Executable):
             conftype = 'Application'
             if not target.gui_app:
                 subsystem = 'Console'
-                entrypoint = 'mainCRTStartup'
         elif isinstance(target, build.StaticLibrary):
             conftype = 'StaticLibrary'
         elif isinstance(target, build.SharedLibrary):
             conftype = 'DynamicLibrary'
-            entrypoint = '_DllMainCrtStartup'
         elif isinstance(target, build.CustomTarget):
             return self.gen_custom_target_vcxproj(target, ofname, guid)
         elif isinstance(target, build.RunTarget):
@@ -1180,8 +1177,6 @@ class Vs2010Backend(backends.Backend):
         if '/ZI' in buildtype_args or '/Zi' in buildtype_args:
             pdb = ET.SubElement(link, 'ProgramDataBaseFileName')
             pdb.text = '$(OutDir}%s.pdb' % target_name
-        if isinstance(target, build.Executable):
-            ET.SubElement(link, 'EntryPointSymbol').text = entrypoint
         targetmachine = ET.SubElement(link, 'TargetMachine')
         targetplatform = self.platform.lower()
         if targetplatform == 'win32':

